experiment/bumpmonitoring/main.go

```diff
@@ -46,19 +51,92 @@ var (
 )
 
 type options struct {
-	srcPath string
-	dstPath string
+	SrcPath string `json:"srcPath"`
+	DstPath string `json:"dstPath"`
+	bumper.Options
+}
+
+func parseOptions() (*options, error) {
+	var config string
+	var labelsOverride []string
+	var skipPullRequest bool
+
+	flag.StringSliceVar(&labelsOverride, "labels-override", nil, "Override labels to be added to PR.")
+	flag.BoolVar(&skipPullRequest, "skip-pullrequest", false, "")
+	flag.Parse()
+
+	var o options
+	data, err := ioutil.ReadFile(config)
+	if err != nil {
+		return nil, fmt.Errorf("read %q: %w", config, err)
+	}
+	if err = yaml.Unmarshal(data, &o); err != nil {
+		return nil, fmt.Errorf("unmarshal %q: %w", config, err)
+	}
+
+	if len(o.SrcPath) == 0 {
+		o.SrcPath = defaultSrcPath
+	}
+	if labelsOverride != nil {
+		o.Labels = labelsOverride
+	}
+	o.SkipPullRequest = skipPullRequest
+	return &o, nil
 }
 
+func validateOptions(o *options) error {
+	if len(o.DstPath) == 0 {
+		return errors.New("dstPath is mandatory")
+	}
+
+	return nil
+}
+
+var _ bumper.PRHandler = (*client)(nil)
+
 type client struct {
-	srcPath string
-	dstPath string
+	SrcPath string
+	DstPath string
```

Comment:
nit: I don't think these need to be exported.

Reply:
good point, done